op-e2e/setup.go

```diff
@@ -235,7 +236,7 @@ type EthInstance interface {
 	WSEndpoint() string
 	HTTPAuthEndpoint() string
 	WSAuthEndpoint() string
-	Close()
+	Close() error
```

Comment:
It's not immediately obvious to me why we changed this signature?

Reply:
It's easier for us to use the same code `node.Close()` to stop both external clients and the geth client started by `geth.Init()`. 

https://github.com/bobanetwork/v3-anchorage/blob/3a9c216e2c6becac6b5553e1f6e8224515c27579/op-e2e/op_geth.go#L144

Reply:
The existing `GethInstance` was already calling `gi.Node.Close()`, if rather than storing the `Node`, we stored the `EthInstance`, we'd get access to that.  But, not a big deal either way.

Reply:
Good suggestion! I replaced `Node` with `EthInstance`.